chapters/101/index.md

```diff
@@ -1,167 +1,113 @@
-# Jina 101: First Thing to Learn About Jina
+# Jina 101
 
-> Checkout [our Github for multi-lingual version of this 101 Guide](https://github.com/jina-ai/jina/blob/master/docs/chapters/101/).
+In Jina 101, you will learn about the basic components of Jina. Before you start, read [the basics of Neural Search](https://jina.ai/2020/07/06/What-is-Neural-Search-and-Why-Should-I-Care.html) to get some context.
 
-## Document & Chunk
+<!-- TOC -->
 
-<span style="display: inline-block; width: 30%;">
 
-![Jina 101 Concept Document and Chunk, Copyright by Jina AI Limited](./img/ILLUS1.png)
+## Document
 
-</span>
+![Document Cartoon](images/document.svg)
 
+Documents are pieces of data in any dataset you want to search, and the input queries you use to find what you want. Basically, they are the input and output data for your search Flows.
```

Comment:
introducing Flow here isnt a little weird? if it has not yet been defined?

Reply:
@JoanFM 
At first, I used `workflow` here but some said it's better to use Flow. I think I'll just change it back.

Reply:
well for me is weird to mame Flow before it being introduced but I am not sure

Reply:
@JoanFM when I'm working on it, I avoid to "name" any component before it's introduced officially. 
There's no way to do it in the perfect way (that fits everyone's preference lol) but I'm changing it to `workflow` here, so we are being consistent throughout 101 & 102 on this logic.